pallets/xcmp-queue/src/lib.rs

```diff
@@ -149,8 +146,8 @@ pub mod pallet {
 	/// case of the need to send a high-priority signal message this block.
 	/// The bool is true if there is a signal message waiting to be sent.
 	#[pallet::storage]
-	pub(super) type OutboundXcmpStatus<T: Config> =
-		StorageValue<_, Vec<(ParaId, OutboundStatus, bool, u16, u16)>, ValueQuery>;
+	pub(super) type OutboundChannelStatus<T: Config> =
```

Comment:
How important is this storage name change?

Will need migrations for this right?

Reply:
Ah right, it does require a migration change... maybe it's time to use the `#[pallet::rename_as]` attribute that I added a while ago?

Reply:
I also didn't check whether a struct is SCALE-encoded the same way as a tuple, so I may have to do a migration anyway.

Reply:
Well ok, I am more confused after reading https://docs.substrate.io/v3/advanced/scale-codec/#data-structures, but since you said that structs encode the same as tuples, I guess it's fine?

Reply:
> I also didn't check whether a struct is SCALE-encoded the same way as a tuple, so I may have to do a migration anyway.

It is the same, but you can write a simple test to confirm this using `encode()` on the old tuple and new struct.